sources/packages/backend/apps/queue-consumers/src/processors/schedulers/student-application-notifications/student-application-notifications.scheduler.ts

```diff
@@ -0,0 +1,62 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { QueueService } from "@sims/services/queue";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../base-scheduler";
+import {
+  InjectLogger,
+  LoggerService,
+  ProcessSummary,
+} from "@sims/utilities/logger";
+import {
+  getSuccessMessageWithAttentionCheck,
+  logProcessSummaryToJobLogger,
+} from "../../../utilities";
+import { QueueNames } from "@sims/utilities";
+import { ApplicationService } from "../../../services";
+import { NotificationService } from "@sims/services/notifications";
+
+@Processor(QueueNames.StudentApplicationNotifications)
+export class StudentApplicationNotificationsScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.StudentApplicationNotifications)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly applicationService: ApplicationService,
+    private readonly notificationService: NotificationService,
```

Comment:
I believe this file can also be created as part of the upcoming PR and can be removed from this PR. Please remove it. Else unwanted declared not used variables will be in the main branch, once merged.